docs/customize/look-and-feel/theme.md

```diff
@@ -27,6 +27,12 @@ The `assets/less/site/globals/site.variables` is where you can override site wid
 @footerDarkColor: /* a shade of your brandColor */;
 ```
 
+To change the orange search button color, in your instance create `assets/less/site/elements/button.variables` file instead of the globals folder and add the following line to it:
+
+```less
+@searchButtonColor: /* your desired color */;
+```
+
```

Comment:
Interesting that this is still needed. I have a note in my codebase saying that having to use `button.variables` could be deleted in v11. Maybe it has to do with something else on our side...

In any case, let's move this below the next paragraph with a little connection so as to get:

---start---

You can override any styling variable in your `site.variables` file. The available styling variables are typically found in the `.variables` files of the various Invenio modules installed (see note below).

For instance, to change the search button color, create the `assets/less/site/elements/button.variables` file (notice the use of the `elements` folder instead of the `globals` one) and add the following line to it:

```less
@searchButtonColor: /* your desired color */;
```

---end---

Reply:
Change is implemented thanks for review @fenekku
I also explored the issue at [InvenioRDM #1642](https://github.com/inveniosoftware/invenio-app-rdm/issues/1642) without success, yet the current approach seems effective. Let's proceed with this solution.